index.js

```diff
@@ -29,11 +29,81 @@ function enterState(newState) {
 start();
 
 async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
+  const welcomeMessage = `109 Ontario Street
+You are standing in a haunting wooded area at dusk. You hear crows cawing and fluttering 
+their wings above. To the left there is a lantern. To the right is a bench. Behind you is a
+church.' 
 There is a door here. A keypad sits on the handle.
 On the door is a handwritten sign.`;
   let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+  console.log('We Offer Sanctuary to Those in Need');
+  process.exit('Sanctuary');
+  break;
+  default;
+  console.log('Sorry, you can not enter.');
+  break;
 }
```

Comment:
This closing curly brace closes off the definition of `start` All the game logic that comes after this isn't actually getting hit because it's outside the function